src/RpcClient/RpcClient.cs

```diff
@@ -245,6 +245,15 @@ public async Task<ContractState> GetContractStateAsync(string hash)
             return ContractStateFromJson((JObject)result);
         }
 
+        /// <summary>
+        /// Queries contract information, according to the contract id.
+        /// </summary>
+        public async Task<ContractState> GetContractStateAsync(int id)
+        {
+            var result = await RpcSendAsync(GetRpcName(), id).ConfigureAwait(false);
+            return ContractStateFromJson((JObject)result);
```

Comment:
Note this doesn't use the "RpcContractState" class which is in the model folder. The function is using the function below it ("ContractStateFromJson", line 257). Should that be changed and updated?

Reply:
Can't use the same method name as `GetContractStateAsync(string hash)`, otherwise it will get `GetContractStateAsync(string hash)` as default when `GetRpcName()`, then id will be and invaild param.

Reply:
It's just a overload for RcpClient and works fine. That's what we want to do. The method that was changed on RpcServer; `GetContractState` to allow contract id to be processed.

**Example**
```csharp
var protocolSettings = ProtocolSettings.Load("config.json", true);
RpcClient rpcClient = new(new("http://127.0.0.1:20332"), "admin", "admin", protocolSettings);
var contractstate = await rpcClient.GetContractStateAsync(742).ConfigureAwait(false);
Console.WriteLine("ScriptHash is {0}.", contractstate.Hash);
Console.WriteLine("Press ENTER to exit . .  .");
Console.ReadLine();
```

Reply:
Sorry, just a fault on my code, it works as the expected!